prow/spyglass/lenses/metadata/lens.go

```diff
@@ -137,6 +146,79 @@ func (lens Lens) Body(artifacts []lenses.Artifact, resourceDir string, data stri
 	return buf.String()
 }
 
+var failedMountRegex = regexp.MustCompile(`MountVolume.SetUp failed for volume "(.+?)" : (.+?)`)
+
+func hintFromPodInfo(buf []byte) string {
+	var report k8sreporter.PodReport
+	if err := json.Unmarshal(buf, &report); err != nil {
+		// This error isn't worth highlighting here, and will come up again
+		// elsewhere.
```

Comment:
Can you explain this more, please? If the `podinfo.json` exists, but is empty or otherwise unparseable that seems like something we should expose. I don't see where this "will come up again elsewhere" and we should probably at least log the error.

Reply:
It is not a critical test failure if the file is unreadable (which is what we're using it for here) — this will be surfaced when the podinfo lens attempts to render.

We can log the error at some log level that we will subsequently never see, but I'm not sure how valuable that actually is.

Reply:
>  this will be surfaced when the podinfo lens attempts to render.

👍 
> We can log the error at some log level that we will subsequently never see, but I'm not sure how valuable that actually is.

Its valuable if we end up hitting the error case unexpectedly, but I agree that seems unlikely and the error will be logged by the podinfo lens if it is enabled. 
/shrug

Reply:
Added a log.